snowcrash.gyp

```diff
@@ -156,6 +156,7 @@
         'test/test-MSONTypeSectionParser.cc',
         'test/test-MSONUtility.cc',
         'test/test-MSONValueMemberParser.cc',
+        'test/test-NamedEndpoint.cc',
```

Comment:
I don't think these test cases should be in a separate file. Please add them in `test-ActionParser.cc`.